docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -339,6 +350,7 @@ Used to lookup fungible assets balances
     * `eip155:8508132` - Scroll
     * `eip155:7854577` - Zora
     * `eip155:1313161554` - Aurora
+* `forceUpdate` - (Optional) List of token contract addresses to force the latest balance update on. `chain` must be also provided for this type of call.
```

Comment:
I've updated the spec and changed back to the GET because we need backward compatibility. Also, I've added the description that the expected format is comma separated list of contract addresses in CAIP-10 format.